Doppler.HtmlEditorApi/Controllers/TemplatesController.cs

```diff
@@ -15,63 +16,19 @@ namespace Doppler.HtmlEditorApi.Controllers
     [ApiController]
     public class TemplatesController : Controller
     {
+        private readonly IRepository _repository;
+
+        public TemplatesController(IRepository Repository)
+        {
+            _repository = Repository;
+        }
+
         [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
         [HttpGet("/accounts/{accountName}/templates/{templateId}")]
         public ActionResult<TemplateModel> GetTemplate(string accountName, int templateId)
         {
-            TemplateModel template = new()
-            {
-                name = "Template Name",
-                counters = new()
-                {
-                    u_row = 1,
-                    u_column = 2,
-                    u_content_text = 1,
-                    u_content_heading = 1,
-                    u_content_menu = 1
-                },
-                body = new()
-                {
-                    rows = Array.Empty<string>(),
-                    values = new()
-                    {
-                        textColor = "#000000",
-                        backgroundColor = "#e7e7e7",
-                        backgroundImage = new()
-                        {
-                            url = "",
-                            fullWidth = true,
-                            repeat = false,
-                            center = true,
-                            cover = false
-                        },
-                        contentWidth = "500px",
-                        contentAlign = "center",
-                        fontFamily = new()
-                        {
-                            label = "Arial",
-                            value = "arial,helvetica,sans-serif"
-                        },
-                        preheaderText = "",
-                        linkStyle = new()
-                        {
-                            body = true,
-                            linkColor = "#0000ee",
-                            linkHoverColor = "#0000ee",
-                            linkUnderline = true,
-                            linkHoverUnderline = true
-                        },
-                        _meta = new()
-                        {
-                            htmlID = "u_body",
-                            htmlClassNames = "u_body"
-                        }
-                    }
-                },
-                schemaVersion = 6
-            };
-
-            return template;
+            var template = _repository.GetTemplateModel(accountName, templateId);
+            return new OkObjectResult(template);
```

Comment:
I do not understand how does it work, I mean, `GetTemplateModel` result is a Task.

Maybe `OkObjectResult<TemplateModel>` accepts a `Task<TemplateModel>` as constructor parameter?

I think that it could be cleaner to use async/await:

```suggestion
            var template = await _repository.GetTemplateModel(accountName, templateId);
            return template;
```

Reply:
Fixed